visualization/app/codeCharta/state/store/metricData/edgeMetricData/edgeMetricData.splitter.ts

```diff
@@ -0,0 +1,6 @@
+import { EdgeMetricDataAction, setEdgeMetricData } from "./edgeMetricData.actions"
+import { EdgeMetricData } from "../../../../codeCharta.model"
+
+export function splitEdgeMetricDataAction(payload: EdgeMetricData[]): EdgeMetricDataAction {
+	return setEdgeMetricData(payload)
+}
```

Comment:
Is there any reason not to call `setEdgeMetricData()` directly? If you definitely want to use an alias, it's possible to do: `export const splitEdgeMetricDataAction = setEdgeMetricData`.

Reply:
No. The file gets auto generated. We should therefore update the template and all existing implementations. But again: I don't want to do this in this PR. Otherwise we'll hit 200-300 changed files.

Reply:
SGTM